lib/Parser/RdfXml.php

```diff
@@ -201,8 +186,28 @@ protected function reify($t, $s, $p, $o, $sType, $oType, $oDatatype = null, $oLa
     }
 
     /** @ignore */
-    protected function startElementHandler($p, $t, $a)
+    public function startElementHandler($parser)
```

Comment:
Can we type-hint `$parser` here? That might make it more clear what it is.

Reply:
Yup, good idea.
It is a bit annoying that the callback has to be public.
But at least @ignore should remove it from the API documentation.